src/App.js

```diff
@@ -1,39 +1,28 @@
-import React, { useEffect } from 'react';
+import React from 'react';
 import { BrowserRouter as Router, Routes, Route } from 'react-router-dom';
-import { useDispatch } from 'react-redux';
 import Body from './components/Body';
 import SignIn from './components/SignIn';
 import SignUp from './components/SignUp';
 import Main from './components/Main';
-import { fetchCars } from './redux/carsSlice';
+import Favourites from './components/Favourite';
+import AddCar from './components/addCar';
+import Detail from './components/Detail';
+import DeleteCar from './components/DeleteCar';
+import './App.css';
 
 function App() {
```

Comment:
- [ ]  Please, kindly refactor this 'App' component and all other functions in your project to ES6 arrow functions, so your code complies with the latest standards.

Refactoring is easy:

```javascript
const App = () => {
  // ...
}
```

## Explanation

The `function` keyword has been soft deprecated in almost all the software industry in favor of ES6 arrow functions. This means, almost nobody uses it anymore, so much that it's considered a non-best practice.

You can do a search in Vscode for the "function" word, and correct as needed.  Shouldn't take you more than 5-10 min so don't worry:  


![image](https://github.com/Rhaegar121/Capstone-Front-End/assets/57421823/5a55da92-375c-4eb5-9c5f-29b18230e892)

That way, your code will meet the latest standards. 💪